app/src/main/java/org/jellyfin/mobile/player/PlayerNotificationHelper.kt

```diff
@@ -84,17 +83,25 @@ class PlayerNotificationHelper(private val viewModel: PlayerViewModel) : KoinCom
                 }
                 setSmallIcon(R.drawable.ic_notification)
                 mediaIcon?.let(::setLargeIcon)
-                setContentTitle(mediaSource.title)
-                mediaSource.artists?.let(::setContentText)
+                setContentTitle(mediaSource.name)
+                //mediaSource.sourceInfo.artists?.let(::setContentText) // TODO
```

Comment:
?

Reply:
I don't have access to the item anymore. Should probably request it through the API when fetching the media source info.

Reply:
Well please do that or remove this line then :p